packages/cli/tests/create-tutorial.test.ts

```diff
@@ -23,9 +30,6 @@ test('cannot create project without installing but with starting', async (contex
   await expect(
     execa('node', [cli, 'create', name, '--no-install', '--start'], {
       cwd: tmpDir,
-      env: {
-        TK_DIRECTORY: baseDir,
-      },
```

Comment:
Now that `TK_DIRECTORY` is not used at all, we could remove it from `packages/cli/src/utils/workspace-version.ts` completely.

Reply:
Oh yeah good point!